### DISCUSSION TITLE
HTML Output from Azure

### DISCUSSION THREAD

**Discussion by abbeb00** [prowler-cloud/prowler#2034] [Q&A]
Hi when I run with 

`./prowler.py azure --browser-auth -M csv html json`

I generate csv and json outputs but no HTML, the csv and json are not super readable either.

Does anyone else experience this issue?

**Comment by toniblyx** [comment]
Hi @abbeb00, thanks for using/testing Prowler. We haven't implimented HTML reports in Azure yet, however you can see the CSV and JSON reports, or the details using `--verbose` like in `prowler azure --browser-auth --verbose`. For more options see `prowler azure --help`
